core/src/main/scala/org/apache/spark/api/r/BaseRRunner.scala

```diff
@@ -82,12 +82,7 @@ private[spark] abstract class BaseRRunner[IN, OUT](
       serverSocket.close()
     }
 
-    try {
-      newReaderIterator(dataStream, errThread)
-    } catch {
-      case e: Exception =>
-        throw new SparkException("R computation failed with\n " + errThread.getLines(), e)
-    }
+    newReaderIterator(dataStream, errThread)
```

Comment:
Is this also related? Looks like the change below are for stderr stuff? This seems to capture exception when create reader iterator.

Reply:
Oh it's not related but here's consistent with Python side. After refactoring, we dont use socket anymore so no external R process related codes run now. I'll double check tomorrow.

Reply:
ok. thanks!

Reply:
Yup, double checked .. should be fine.